.github/release.yml

```diff
@@ -0,0 +1,19 @@
+changelog:
+  exclude:
+    labels:
+      - ignore-for-release
+    authors:
+      - octocat
+  categories:
+    - title: Breaking Changes 🛠
```

Comment:
🔥
or
🚨
?
(otherwise its too similar to Fixes)

Reply:
done!